app/models/concerns/tagable.rb

```diff
@@ -6,17 +6,36 @@ module Tagable
     has_many :tags, through: :taggings
   end
 
-  TAGABLE_CLASSES = [Entity, List, Relationship]
+  class_methods do
+    def category_str
+      name.downcase.pluralize
+    end
+
+    def category_sym
+      category_str.to_sym
+    end
+  end
 
-  # sorts a list of tagables in descending order of relationships to tagables w/ same tag
-  def self.sort_by_related_tagables(tagables)
-    tagables
+  # () -> Array[ClassConstant]
+  def self.classes
```

Comment:
cool! and Tagable.classes is shorter and nicer looking than the uppercase constant

Reply:
huzzah! :D